packages/shared/src/hooks/useOrganizations.tsx

```diff
@@ -13,8 +13,8 @@ type UseOrganizationsReturn =
       createOrganization: undefined;
 
       /**
-       * @deprecated Use `membershipList` from `useOrganization`
-       * Example const {membershipList} = useOrganization()
+       * @deprecated Use `memberships` from `useOrganization`
+       * Example const {memberships} = useOrganization()
```

Comment:
❓ Should this text be formatted a little easier to read?
```suggestion
       * Example: `const {memberships} = useOrganization()`
```